test/contrib/automotive/gmlan_trace.candump

```diff
@@ -1,1087 +0,0 @@
-(0.297000) vcan2 12A#122010FF00001080
```

Comment:
It's a bummer this even got into here in the first place.. I'll try to be much more careful reviewing stuff from now on